configure

```diff
@@ -84,6 +84,7 @@ include_package_wavefront="wavefront"
 include_package_weaver="weaver"
 
 swig_bindings=""
+workqueue_swig_bindings=""
```

Comment:
Could you generate the bindings for chirp too? That way we don't have to manage two different swig configurations.

Reply:
Will do